src/v/cloud_storage/remote_partition.cc

```diff
@@ -546,7 +546,7 @@ remote_partition::get_term_last_offset(model::term_id term) const {
       "The manifest for {} is not expected to be empty",
       _manifest.get_ntp());
     if (last->segment_term == term) {
-        return last->committed_kafka_offset();
+        return last->next_kafka_offset() - kafka::offset(1);
```

Comment:
This can also be off by one in previous versions, right? I see this gets returned in the `OffsetForleaderEpochResponse`, but I don't know if it's a big deal or not.

Reply:
Right